chapters/synchronous.tex

```diff
@@ -444,8 +444,9 @@ \section{Clock Constructors}\doublelabel{clock-constructors}
 \textbf{Clock with Real Interval}\\
 
 The input argument, \lstinline!interval!, is a clocked Component Expression (see
-\autoref{argument-restrictions-component-expression}) or a parameter expression of type \textbf{Real} with
-min=0.0 and unit ``s''. The output argument is of base type Clock that
+\autoref{argument-restrictions-component-expression}) or a parameter expression.
+Interval must be strictly positive (\lstinline!interval>0.0!) of type \textbf{Real} unit ``s''. 
```

Comment:
Having lstinline instead of textbf would likely be better, but that wasn't changed in this commit - so seems that missing cleanup of synchronous section. However, that will take a bit more care since boldface is used for multiple purposes in that chapter.